docs/guides/lint-arazzo.md

```diff
@@ -120,6 +120,16 @@ To expand the linting checks for an Arazzo description, start by enabling
 some of the built-in rules. The currently-supported rules are:
 
 - `parameters-no-body-inside-in`: the `in` section inside `parameters` must not contain a `body`.
+- `strict-source-description-type`: the `type` property of the `sourceDescription` object must be either `openapi` or `arazzo`.
+- `arazzo-version-enum`: the `version` property must be one of the allowed values: [`1.0.0`].
+- `workflow-workflowId-unique`: the `workflowId` property must be unique across all workflows.
+- `step-stepId-unique`: the `stepId` must be unique amongst all steps described in the workflow.
+- `sourceDescription-name-unique`: the `name` property of the `sourceDescription` object must be unique across all source descriptions.
+- `workflow-dependsOn-unique`: the items in the `workflow` `dependsOn` property must be unique.
```

Comment:
Does this rule also include checking if the `workflowId` exists? If yes, maybe it's better to rename the rule into something more generic, like `workflow-dependsOn`?

Reply:
Its opposite, the current workflow dependsOn execution of the list of workflows.
`A list of workflows that MUST be completed before this workflow can be processed.`
We don't support async execution - for for our implementation we will execute all workflows in this list before running the current workflow.
It has type unique - so it make sense to check it.
All rules here are based on spec description and suggested JSON schema.